script/deploy.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -uexo pipefail
+
+${GPG:-gpg2} --detach-sign install.sh
```

Comment:
So this still means you are the only one who can sign this? We should probably think about expanding our keyring...

Reply:
The D language should be responsible for this, all passwords and similar. So we don't depend on a single person.